File: PythonDatabaseCompatibility/dbCredentials.py
import json
import os


class DbCredentials:
    
    def __init__(self):
        ''' Currently set up to use VCAP
        but may need to adapt to use json line
        '''
        dbinfo = json.loads(os.environ['VCAP_SERVICES'])['sqldb'][0]
        dbcred = dbinfo["credentials"]
        self.db = dbcred['db']
        self.hostName = dbcred['hostname']
        self.port = dbcred['port']
        self.uid = dbcred['username']
        self.pwd = dbcred['password']
        
                     

File: RubyDatabaseCompatibility/dbCredentials.rb
require 'json'

class DbCredentials
	attr_accessor :db, :hostname, :port, :uid, :password

	def initialize
		vcap_services = JSON.parse(ENV('VCAP_SERVICES'))
		dbcred = vcap_services["credentials"]
		@db = dbcred["db"]
		@hostname = dbcred["hostname"]
		@port = dbcred["port"]
		@uid = dbcred["username"]
		@pwd = dbcred["password"]











